packages/contracts/package.json

```diff
@@ -25,7 +25,7 @@
   "devDependencies": {
     "@types/node": "^18.15.11",
     "ds-test": "https://github.com/dapphub/ds-test.git#e282159d5170298eb2455a6c05280ab5a73a4ef0",
-    "forge-std": "https://github.com/foundry-rs/forge-std.git#74cfb77e308dd188d2f58864aaf44963ae6b88b1",
+    "forge-std": "github:foundry-rs/forge-std",
```

Comment:
This makes my `pnpm install` fail since this defaults to ssh I don't have that setup on this machine.
Is there a reason to use it way or can we go back to `"forge-std": "https://github.com/foundry-rs/forge-std.git#74cfb77e308dd188d2f58864aaf44963ae6b88b1",`?